Unicode/is_Ligature.c

```diff
@@ -1,104 +1,21 @@
 /*
 Copyright: 2012 Barry Schwartz
+    Create is_Ligature.c (using a python script) to test for Vulgar fractions
 Copyright: 2016 Joe Da Silva
-License: BSD-3-clause
+    Re-write is_Ligature.c to test for ligatures, Vulgar and Other-fractions.
+Copyright: 2016 Gioele Barabucci
+    Simplify makeutype.c code. Split is_Ligature.c, create is_Ligature_data.h
+License: GPL-3+
```

Comment:
Thanks! I do appreciate that.

As to the more general issue, I wasn't equating copyright and licensing. My point was that since we don't have a means to consolidate copyright ownership/control (which would allow one party to issue the code under new licenses in the future), the only way for us to retain licensing and relicensing flexibility is to use permissive/flexible licenses.